single-function/lru-cache/submission.js

```diff
@@ -0,0 +1,66 @@
+class DoubleLinkedList {
+  constructor(key, value) {
```

Comment:
Since we are being tidy with a class rather than an object literal, may as well accept an optional nextNode in the constructor and save repeated manual next/prev wireup by users of the class.

Reply:
I can't seem to do this and have my tests pass. I will keep trying, but I might want to pair with you for a bit if I can't nail it down.